src/Template/Layout/js/app/components/filter-box.js

```diff
@@ -73,6 +73,15 @@ export default {
                         </select>
                     </span>
 
+                    <span v-else-if="filter.type === 'checkbox'">
+                        <label>Both</label>
```

Comment:
`Any` instead of `Both` ? 

We should also use gettext localization, so something like `{{ __('Yes') }}` - how do we do it here? 
Maybe resolve this in a new PR

Reply:
right, we do it with attributes for now.
we can do it in other ways also. should we create a refactor issue?

Reply:
yes, create a refactor issue to localize `js` strings via gettext